### ISSUE TITLE
Address issues where Testem suite does not exit after all tests are run

### ISSUE THREAD

**Issue by hectcastro** [WikiWatershed/model-my-watershed#3110]
In https://github.com/WikiWatershed/model-my-watershed/pull/3108 it was revealed that occasionally the test suite does not exit after completion. We should make sure to remedy this so that CI based deployments are not blocked by failed `develop` builds.

**Comment by rajadain** [comment]
I have fiddled with the versions of the following:

* xvfb
* phantomjs
* testem
* mocha

and the behavior does not change. I even upgraded node on the `app` vm to see if that helped, and it did not. If I run testem directly (without xvfb):

```
vagrant@app:/var/www/mmw/static$ /opt/app/node_modules/.bin/testem -f /opt/app/testem.json ci Firefox
```

I get the same hanging behavior. I tried both with and without the `$*` parameter.

At this point I recommend we turn off the tests from CI for now, and make a card to upgrade the testing infrastructure to something more modern (PhantomJS for example has been [abandoned](https://github.com/ariya/phantomjs/issues/15344) since March 2018).
